utils/Balance.ts

```diff
@@ -0,0 +1,129 @@
+import { GenericCoin } from "@/types";
+import { Codec, Balance as ApiBalance } from "@cennznet/types";
+import Big, { BigSource, RoundingMode } from "big.js";
+import BN from "bn.js";
+
+interface AsBalanceOptions {
+	withSymbol: boolean;
+}
+
+type BalanceDescriptor = Pick<GenericCoin, "symbol" | "decimals">;
```

Comment:
how does `Pick` work?

Reply:
`Pick` is another TS utility type (AKA magic stuff), it returns a new type that only consist type of the keys (`"symbol" | "decimals"`) defined in `GenericCoin`

See here for more of those utility types https://www.typescriptlang.org/docs/handbook/utility-types.html